webpack.mix.js

```diff
@@ -0,0 +1,86 @@
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const mix = require('laravel-mix');
+const { escapeRegExp } = require('lodash');
+const { sep } = require('path');
+
+/*
+ |--------------------------------------------------------------------------
+ | Mix Asset Management
+ |--------------------------------------------------------------------------
+ |
+ | Mix provides a clean, fluent API for defining some Webpack build steps
+ | for your Laravel application. By default, we are compiling the Sass
+ | file for the application as well as bundling up all the JS files.
+ |
+ */
+
+mix.setPublicPath('../glutenblocks');
+
+mix
+    .options({
+        processCssUrls: false,
+        lang: 'sass',
+    })
+    .webpackConfig({
+        devtool: '#source-map',
+        output: {
+            chunkFilename: 'build/[name].js?id=[chunkhash]',
+            publicPath: '/app/plugins/glutenblocks/',
```

Comment:
It doesn't seems to be a robust way to do so. A normal WordPress installation will install this to `/wp-content/plugins/glutenblocks/`

Reply:
Public Path was removed since it's not used.